src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/BreakpointExtensions.cs

```diff
@@ -73,7 +74,15 @@ public static StackdriverBreakpoint Convert(this Breakpoint breakpoint)
                 EvaluatedExpressions =
                 {
                     breakpoint.EvaluatedExpressions?.Select(variable => variable.Convert()).ToList()
-                }
+                },
+
+                Expressions =
+                {
+                    breakpoint.EvaluatedExpressions?.Select(variable => variable.Name).ToList()
```

Comment:
Why do we need this here? This seems out of place.  Is this something we missed in the past?  If so should we have a test somewhere for this?